path_config.go

```diff
@@ -76,8 +76,23 @@ func (b *backend) configFields() map[string]*framework.FieldSchema {
 }
 
 func (b *backend) configCreateUpdateOperation(ctx context.Context, req *logical.Request, fieldData *framework.FieldData) (*logical.Response, error) {
```

Comment:
Is there a difference between PUT/POST at the API level?

Right now a create operation would retain existing values if the plugin config already exists; whereas as a user when I call explicitly a create operation I expect APIs to either throw an error or set the config as it was defined without keeping values implicitly depending on if overriding is allowed.

If create/update use the same commands/api calls then this comment doesn't apply since users can't choose between the 2 operations.